packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/launch-modal.scss

```diff
@@ -0,0 +1,71 @@
+// Common styles for web modal and mobile page overlay
+.launch-modal-styles {
+    text-align: center;
+    .modal-content {
+        @include mobile {
+            height: 100%;
+            justify-content: center;
+            display: flex;
+            flex-direction: column;
+        }
+        h1 {
+            font-size: 2rem;
+            font-weight: 700;
+            line-height: 3rem;
+        }
+
+        p {
+            font-size: 16px;
```

Comment:
Font styles should be set through props in `Text` component we have.

Reply:
And usually we try to avoid using tag selectors